### ISSUE TITLE
TODO: Hard Sigmoid

### ISSUE THREAD

**Issue by canyalniz** [canyalniz/CNN-Inference-Didactic#8]
[Core Keras Activation Hard Sigmoid](https://keras.io/activations/#hard_sigmoid) must be implemented and documented.
